community/rust/APKBUILD

```diff
@@ -161,6 +164,17 @@ lldb() {
 	_mv "$pkgdir"/$_sharedir/etc/lldb_*.py $_sharedir/etc/
 }
 
+src() {
+	pkgdesc="Rust source code for ycmd and racerd code completion support"
+	depends="$pkgname"
+	license="$license OFL-1.1 GPL-3.0-or-later GPL-3.0-with-GCC-exception CC-BY-SA-3.0 LGPL-3.0"
+
+	mkdir -p "$subpkgdir"/usr/share/rust
+	cd "$subpkgdir"
+
+	_mv "$srcdir"/src usr/share/rust/
```

Comment:
Please install sources into `/usr/src/rust` instead of `/usr/share/rust`.

Reply:
The final result is /usr/share/rust/src.  I will unload the contents of src in there (/usr/src/rust).